corehq/apps/domain/templates/domain/admin/dhis2/log_detail.html

```diff
@@ -0,0 +1,61 @@
+{% extends "domain/admin/info_base.html" %}
+{% load hq_shared_tags %}
+{% load hqstyle_tags %}
+{% load i18n %}
+
+{% block page_content %}
+<form>
+    <fieldset>
+        <legend>{% trans "DHIS2 Logs" %}</legend>
+        <div>
+            <table class="table table-striped">
+                <tr>
+                    <th>{% trans "Timestamp" %}:</th>
+                    <td>{{ log.timestamp }}</td>
+                </tr>
+                <tr>
+                    <th>{% trans "Request" %}:</th>
+                    <td>{{ log.request_method }} {{ log.request_url }}</td>
+                </tr>
+                <tr>
+                    <th>{% trans "Status" %}:</th>
+                    <td>{% if log.response_status %}{{ log.response_status }}{% else %}[N/A]{% endif %}</td>
+                </tr>
+
+                <tr>
+                    <th>{% trans "Request headers" %}:</th>
+                    <td><pre>{{ log.request_headers }}</pre></td>
+                </tr>
+                {% if log.request_params %}
+                <tr>
+                    <th>{% trans "Request params" %}:</th>
+                    <td><pre>{{ log.request_params }}</pre></td>
+                </tr>
+                {% endif %}
+                {% if log.request_body %}
```

Comment:
I'd prefer to just leave these blank instead of completely removing them from the UI